cms/envs/common.py

```diff
@@ -2230,3 +2230,6 @@
 # .. toggle_tickets: ARCH-1253
 # .. toggle_status: supported
 DISABLE_DEPRECATED_SIGNUP_URL = False
+
+##### LOGISTRATION RATE LIMIT SETTINGS #####
+LOGISTRATION_RATELIMIT_RATE = '500/5m'
```

Comment:
I think default value should be low e.g **5/m** and we should set **500/m** in edx-vars.

Reply:
@adeelehsan That will again kill the e2e tests on stage. Previously we set a very low limit and e2e tests started failing and we had to revert that.

Here is the revert PR https://github.com/edx/edx-platform/pull/24052/files.